samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { app, monetization } from "@microsoft/teams-js";
+import { Text, Button, Flex } from '@fluentui/react-northstar'
+import './tab.css'
+
+const Tab = () => {
+
+    React.useEffect(() => {
+        app.initialize();
+    }, [])
+
+
+    var planInfo = {
+        planId: "<Plan id>", // Plan Id of the published SAAS Offer
+        term: "<Plan Term>" // Term of the plan.
```

Comment:
What value we need to add it here, please specify in readme, if not done ?

Reply:
Yes we have added in the comments what value to be added